### ISSUE TITLE
Complains about lack of keys when running without https

### ISSUE THREAD

**Issue by timbl** [linkeddata/ldnode#303]
Command line:
```
/usr/local/bin/node /devel/github.com/linkeddata/ldnode/bin/ldnode.js  --verbose \
    --root /Library/WebServer/Documents/  \
    --no-file-browser \
    --strict-origin \
    --error-pages /Users/timbl/Documents/Automation/server/error \
    --proxy /xss -p 3080
```
log:
```
Wed, 30 Mar 2016 21:45:06 GMT ldnode:settings SSL Private Key path: undefined
Wed, 30 Mar 2016 21:45:06 GMT ldnode:settings SSL Certificate path: undefined
Missing SSL cert and SSL key to enable WebIDs
Error: Missing SSL cert and SSL key to enable WebIDs
```

**Comment by nicola** [comment]
@timbl try with --no-webid

**Comment by timbl** [comment]
I wonder whether —webid became a default on attribute
@timbl heads into the GDS

timbl 10:11
yes
   // Webid to be default in command line
  if (argv.webid !== false) {
    argv.webid = true
  }
that would do it

timbl 10:19
That should be changed in the doc
you just need —no-webid

timbl 10:32
Sigh … now "Solid server (ldnode v0.2.26) running on https://localhost:3456/
“ who said anything about 3456 when I said -p 3080 ????

timbl 11:11
So the -p short form has been removed

**Comment by timbl** [comment]
Yes @nicola you now need --no-webid    and --port instead of -p
-p should have thrown an "unrecognized arg" error really.

**Comment by timbl** [comment]
Todo: Fix command line documentation 
todo: Fix github home page doc https://github.com/linkeddata/ldnode

**Comment by timbl** [comment]
Todo:  Error when unrecognized command line arg

**Comment by nicola** [comment]
perfect @timbl , does this fix: #305 !

webid did become true by default and we dropped short flags `-.`
